rqt_py_common/src/rqt_py_common/extended_combo_box.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
```

Comment:
@dirk-thomas Can you confirm that these lines can be deleted for all the files that aren't executable